src/ontology/mondo-edit.obo

```diff
@@ -359661,12 +359661,14 @@ property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/
 [Term]
 id: MONDO:0014946
 name: Sifrim-Hitz-Weiss syndrome
+def: "A rare multiple congenital anomalies/dysmorphic syndrome due to CHD4 gene mutations. It is characterized by developmental delay, speech delay and variable degree of intellectual disability (mostly mid-to-moderate but some patients may also have normal intelligence). Even though clinical manifestations are significantly variable among patients, most patients manifest dysmorphic facial features (could sometimes include macrocephaly), congenital heart defects, hypotonia and opthalmologic abnormalities. Other clinical features may include brain structure anomalies, skeletal anomalies, hearing impairment and hypogonadism (only in males)." [Orphanet:653712]
 subset: gard_rare {source="MONDO:GARD"}
 subset: nord_rare {source="MONDO:NORD"}
 subset: ordo_disorder {source="Orphanet:653712"}
 subset: orphanet_rare {source="Orphanet:653712"}
 subset: otar {source="MONDO:OTAR"}
 subset: rare
+synonym: "CHD4-related neurodevelopmental disorder" EXACT [https://orcid.org/0000-0001-9310-0163, Orphanet:653712]
 synonym: "Sifrim-Hitz-Weiss multiple congenital anomalies-intellectual disability syndrome" RELATED []
 synonym: "Sifrim-Hitz-Weiss multiple congenital anomalies-mental retardation syndrome" RELATED DEPRECATED []
 synonym: "Sifrim-Hitz-Weiss syndrome" EXACT [DOID:0070529, OMIM:617159, Orphanet:653712]
```

Comment:
can you please add the term tracker item to this as well? It's good practice to add a link to the GitHub ticket when new changes are made to a new, thanks!